src/middlewared/middlewared/schema.py

```diff
@@ -172,9 +172,9 @@ def validate(self, value):
 
 class IPAddr(Str):
 
-    def validate(self, value):
+    def validate(self, value, cidr=True):
```

Comment:
This should be passed in constructor so we could create `IPAddr()` field or `IPAddr(cidr=True)` field.